test/models/binomial.py

```diff
@@ -0,0 +1,72 @@
+from typing import Union
+from scipy import stats
+from scipy.special import logit, expit as inv_logit, log1p
+import numpy as np
+import numpy.typing as npt
+
+
+class Binomial:
+    """
+    Binomial model with a (conjugate) beta prior.
+
+    This model is decomposed as a prior and a likelihood for
+    testing with samplers that require that, e.g. SMC. The
```

Comment:
"e.g." always comes with a comma, and is usally parenthetical, so that'd be "(e.g., SMC)".